### ISSUE TITLE
Update to support Minecraft 1.20.4 Fabric

### ISSUE THREAD

**Issue by lospejos** [LambdAurora/LambdaBetterGrass#100]
Please update to support Minecraft 1.20.4 Fabric. Thanks!

**Comment by Zerogoki00** [comment]
+1

**Comment by mucookul** [comment]
Currently LambdaBetterGrass uses Quilt's APIs but Quilt hasn't been updated for 1.20.4 yet, the Quilt Team is working on it but it might take a while.

**Comment by notrohit1702** [comment]
1.20.2 is out for QFAPI/QSL now. (in alphas but I think alpha 3 is quite stable)

**Comment by Pouri-Shoon** [comment]
Update for 1.20.1 plz

**Comment by notrohit1702** [comment]
> Update for 1.20.1 plz

Isn't there already a version for 1.20.1?

**Comment by ChristopherTheBoss** [comment]
Dropping Fabric support for unstable software is so stupid.
Maybe I'm stupid because I've never successfully launched my fully modded Minecraft with Quilt. I've managed it at the bare minimum, haven't bothered with a binary search.
In fact, with 1.20.2 Quilt launches with the base of mods but fails to launch with LambdaBetterGrass.
For me this feature is essential for immersion in the game, and the developer doesn't seem to be active. I can't blame them I started scripting for a niche website and it's quite time consuming, especially with my other commitments IRL.
No successor seems to have stepped in for LBG I'll try looking through forks again I guess.
If you can't commit to this project, why don't you look for another developer? IDK this has been the source of continuous frustration for a year (?)

**Comment by MimzyFang** [comment]
If you are having issues launching quilt please join the quilt discord and post your logs in quick support or a support forum thread 

The people on there are very nice and helpful

**Comment by LambdAurora** [comment]
> If you can't commit to this project, why don't you look for another developer? IDK this has been the source of continuous frustration for a year (?)

I actually can, though lately I lacked the motivation to work on this project due to Quilt's issues with updating recently and due to the fact there is no other APIs that satisfies my needs with low maintenance needs.
At this point it's clear Mojang will do regular internal refactoring in minor versions and it will be terrible for mod devs because now we need to maintain our mods even more through tedious and not particularly fun updates.

I'd appreciate if this community would stop chasing after the latest available version because it's tiring to maintain multiple mods. I'd rather spend my time adding new cool features.

Also I'd appreciate if you didn't call my decision stupid, the "unstable software" so far has one of the most complete resource loader implementation in the modding ecosystem. I plan on doing something to avoid being stuck due to a lack of updates but time is a precious resource.

And as much as I appreciate the "delegate to someone else maintenance", I frankly do not trust anyone for the resource loading code.

**Comment by MimzyFang** [comment]
I fully agree with @LambdAurora here there is absolutely no point in upgrading to anything newer than 1.20.1 at this time as that version is absolutely fine and the versions after don't fix any critical vulnerabilitys or add cool new gameplay changes and a lot of other mods see it like that as well and are still on 1.20.1 and while 1.20.5 will add some small cool additions these could probably just be backported by a small mod so until 1.21 comes out there is no point in going through the hassle updating to a newer version

**Comment by KennyDevT** [comment]
>  No successor seems to have stepped in for LBG I'll try looking through forks again I guess.

SuperBetterGrass Works on 1.20.4

**Comment by ItsRodrigoAl** [comment]
So... DynamicLights have support for 1.20.4 but BetterGrass dont? whats the reason lol

**Comment by LambdAurora** [comment]
Since then, 1.20.4 is a minor version that is for the most part abandoned, such this issue is no longer relevant, especially with other port request issues.
